spec/jobs/sufia_resque_queue_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe Sufia::Resque::Queue do
+  let (:subject) { Sufia::Resque::Queue.new "test_queue" }
+  let (:job) { double('job') }
+
+  context "no errors" do
+    it "queues the job" do
+      expect(::Resque).to receive(:enqueue_to).and_return(true)
+      subject.push(job)
+    end
+  end
+
+  context "timeout error" do
+    it "retries on timeout" do
+      expect(::Resque).to receive(:enqueue_to).and_raise(Redis::TimeoutError)
```

Comment:
@cam156 the syntax is A-OK!

Why would the job pushed on line 18 be expected to raise an error if the job on line 10 didn't?  I know the tests pass but I have a sinking suspicion here that the tests may not be doing what you want them to do.